jax/_src/api.py

```diff
@@ -228,15 +229,16 @@ def _infer_argnums_and_argnames(
 
 
 def jit(
-  fun: Callable,
```

Comment:
These are just indentation changes, not relevant to the main content of the PR. (Sorry for including them...)